### DISCUSSION TITLE
Can we user a local payment provider other than stripe

### DISCUSSION THREAD

**Discussion by Tdnshah** [HiEventsDev/Hi.Events#250] [General]
I would like to add a custom payment provider available in my country, does hi.events provides this feature?
I can contribute back this feature if not available already with hi.events.

**Comment by daveearley** [comment]
Currently, we only support Stripe, however, the ability to add other providers will be added in the not too distant future. What provider in particular would you want to add?

**Reply by jpiston** [reply]
Hi daveearley . 
I'd kind same similar question. I would like to know if I can implement a "PAID TICKETS" order but on hold.  The aim is to allow the user to perform the payment using the local bank apps and upon verification and validate from my part. Then issue the ticket.

Do you thing such thing can be done?

**Reply by islandmix** [reply]
Your scenario is an offline payment scenario. That request already exists here: https://github.com/HiEventsDev/Hi.Events/issues/231

Definitely not something that is scaleable though.

**Reply by drd2aiki** [reply]
> Currently, we only support Stripe, however, the ability to add other providers will be added in the not too distant future. What provider in particular would you want to add?

I am a reseller for Shift4 and would love to process through them. My customers are mostly small independent restaurants and many are looking for event management.

**Reply by FourHeadedDev** [reply]
Got it. Since I’m based in India, I’d really like to see Razorpay support — it’s one of the most widely used payment providers here.

**Comment by FiftyBugsOfGrey** [comment]
Stripe is not easy to use in India. Would be nice to add other options like RazorPay, PayU, Paypal etc. attendize supports a wide range of payment processors, and if hi.events could do that, it would be great.

**Reply by FourHeadedDev** [reply]
yeah same issue bro have you setup hi.events for india ever ?
then i need a bit help from you

i am also from india

**Comment by donjuanhws** [comment]
Hi Guys, excellent platform, quickly want to check-in and get a progress update on supporting other payment providers, as the code stands at the moment it is not easy to integrate / extend existing code to accommodate for other online payment providers. I read through the Issues and found this comment [https://github.com/HiEventsDev/Hi.Events/issues/231#issuecomment-2564292384](url) and just wanted to check in on progress here because Stripe is also not supported in our Country and need to expand the payment gateway component to work with one of our local providers.

**Comment by FourHeadedDev** [comment]
I would like to see Razorpay in it
In india Razorpay is widely used
Razorpay is stripe of India
